pulp_smash/tests/ostree/api_v2/test_copy.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+"""Test copy of specific content from/to OSTree repositories."""
+import unittest
+from urllib.parse import urljoin
+
+from pulp_smash import api, config, utils
+from pulp_smash.constants import OSTREE_BRANCH, OSTREE_FEED, REPOSITORY_PATH,\
+    OSTREE_FEED_SMALL
+from pulp_smash.tests.ostree.utils import gen_distributor, gen_repo
+from pulp_smash.tests.ostree.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+
+class CopyTestCase(unittest.TestCase):
+    """Test copy of specific content from/to OSTree repositories."""
+
+    def test_all(self):
+        """Test copy of specific content from/to OSTree repositories.
+
+        Verify that:
+
+        * When doing a copy from one OSTree repo to another, just certain units
```

Comment:
Nitpick: When there's only one item in an unordered list, there's little reason for the unordered list to exist. This could be written as:

> Verify that filters can be used to restrict which units are copied between repositories.